webapp/src/app/pages/components/knowledge-graph/cytoscape/cytoscape.component.ts

```
@@ -320,13 +369,17 @@
               .style('display', 'none');
           }
           let nodesToHide = nodes.filter(function (e: any) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'nodesToHide' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)